api/v1alpha1/driver_types.go

```diff
@@ -294,7 +294,7 @@ type DriverSpec struct {
 
 	// Select a policy for snapshot behavior: none, autodetect, snapshot, sanpshotGroup
 	//+kubebuilder:validation:Optional
-	//+kubebuilder:validation:Enum:=none;autodetect;volumegroupsnapshot;volumesnapshot
+	//+kubebuilder:validation:Enum:=none;autodetect;volumeGroupSnapshot;volumeSnapshot
```

Comment:
@subhamkrai I believe `autodetect` needs to be removed from the validation as it is not a valid value anymore.
Can you please add this change to the PR?

Reply:
done